ownCloud/Resources/Theming/Branding.plist

```diff
@@ -0,0 +1,546 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>canAddAccount</key>
+	<false/>
+	<key>feedbackMail</key>
+	<string>support@owncloud.online</string>
+	<key>Bundle</key>
+	<dict>
+		<key>organizationLogoName</key>
+		<string>bookmark-logo</string>
+		<key>organizationBackgroundName</key>
+		<string>brand-background.jpg</string>
+		<key>organizationName</key>
+		<string>ownCloud.online</string>
+	</dict>
+	<key>Profiles</key>
+	<array>
+		<dict>
+			<key>identifier</key>
+			<string>ownCloud.online</string>
+			<key>name</key>
+			<string>ownCloud.online</string>
+			<key>welcome</key>
+			<string>Welcome to ownCloud.online</string>
+			<key>prompt</key>
+			<string>Enter your username and password:</string>
```

Comment:
Please can you point me, which keys do you mean?

Reply:
I thought of the `prompt` key in the Branding.plist. Since it's just a quick change, I made the necessary changes (committing shortly). Now, there's `prompt`, which is used by default. In addition, `promptForPasswordAuth` and `promptForTokenAuth` allow providing a specific prompt for password authentication and token authentication.